<!doctype html>
<html class="no-js" lang="en">
<head>
	<meta charset="utf-8">

	<meta name="viewport" content="width=device-width, initial-scale=1.0">
	<title></title>

    <link rel="stylesheet" type="text/css" href="http://localhost:4001/assets/css/styles_feeling_responsive.css">

    <link href="/assets/css/fontawesome.all.min.css" rel="stylesheet">

  

	<script src="http://localhost:4001/assets/js/modernizr.min.js"></script>

	<script src="https://ajax.googleapis.com/ajax/libs/webfont/1.5.18/webfont.js"></script>
	<script>
		WebFont.load({
			google: {
				families: [ 'Zalando+Sans:400,500,600,700:latin', 'Trocchi:400,700:latin' ]
			}
		});
	</script>

	<noscript>
		<link href='http://fonts.googleapis.com/css?family=Zalando+Sans:400,500,600,700|Trocchi:400,700' rel='stylesheet' type='text/css'>
	</noscript>


	<!-- Search Engine Optimization -->
	<meta name="description" content="">
	
	
	
	
	
	
	<link rel="canonical" href="http://localhost:4001/software/">


	<!-- Facebook Open Graph -->
	<meta property="og:title" content="">
	<meta property="og:description" content="">
	<meta property="og:url" content="http://localhost:4001/software/">
	<meta property="og:locale" content="en_EN">
	<meta property="og:type" content="website">
	<meta property="og:site_name" content="The Jackson Lab">
	
	


	

	<link type="text/plain" rel="author" href="http://localhost:4001/humans.txt">

	

	

	<link rel="icon" sizes="32x32" href="http://localhost:4001/assets/img/favicon-32x32.png">

	<link rel="icon" sizes="192x192" href="http://localhost:4001/assets/img/touch-icon-192x192.png">

	<link rel="apple-touch-icon-precomposed" sizes="180x180" href="http://localhost:4001/assets/img/apple-touch-icon-180x180-precomposed.png">

	<link rel="apple-touch-icon-precomposed" sizes="152x152" href="http://localhost:4001/assets/img/apple-touch-icon-152x152-precomposed.png">

	<link rel="apple-touch-icon-precomposed" sizes="144x144" href="http://localhost:4001/assets/img/apple-touch-icon-144x144-precomposed.png">

	<link rel="apple-touch-icon-precomposed" sizes="120x120" href="http://localhost:4001/assets/img/apple-touch-icon-120x120-precomposed.png">

	<link rel="apple-touch-icon-precomposed" sizes="114x114" href="http://localhost:4001/assets/img/apple-touch-icon-114x114-precomposed.png">

	
	<link rel="apple-touch-icon-precomposed" sizes="76x76" href="http://localhost:4001/assets/img/apple-touch-icon-76x76-precomposed.png">

	<link rel="apple-touch-icon-precomposed" sizes="72x72" href="http://localhost:4001/assets/img/apple-touch-icon-72x72-precomposed.png">

	<link rel="apple-touch-icon-precomposed" href="http://localhost:4001/assets/img/apple-touch-icon-precomposed.png">	

	<meta name="msapplication-TileImage" content="http://localhost:4001/assets/img/msapplication_tileimage.png">

	<meta name="msapplication-TileColor" content="#fabb00">


	

</head>
<body id="top-of-page" class="page-fullwidth">
	
	<div id="navigation" class="sticky">
  <nav class="top-bar" role="navigation" data-topbar>
    <ul class="title-area">
      <li class="name">
      <h1 class="show-for-small-only"><a href="http://localhost:4001" class="icon-tree"> The Jackson Lab</a></h1>
    </li>
       <!-- Remove the class "menu-icon" to get rid of menu icon. Take out "Menu" to just have icon alone -->
      <li class="toggle-topbar menu-icon"><a href="#"><span>Nav</span></a></li>
    </ul>
    <section class="top-bar-section">

      <ul class="right">
        

              

          
          
        

              

          
          
        

              

          
          
        

              

          
          
        

              

          
          
        

              

          
          
        

              

          
          
            
            
              <li class="divider"></li>
              <li><a  href="http://localhost:4001/contact/">Contact</a></li>

            
            
          
        
        
      </ul>

      <ul class="left">
        

              

          
          

            
            
              <li><a  href="http://localhost:4001/">Home</a></li>
              <li class="divider"></li>

            
            
          
        

              

          
          

            
            
              <li><a  href="http://localhost:4001/team/">Team</a></li>
              <li class="divider"></li>

            
            
          
        

              

          
          

            
            
              <li><a  href="http://localhost:4001/research/">Research</a></li>
              <li class="divider"></li>

            
            
          
        

              

          
          

            
            
              <li><a  href="http://localhost:4001/publications/">Publications</a></li>
              <li class="divider"></li>

            
            
          
        

              

          
          

            
            

              <li class="has-dropdown">
                <a  href="http://localhost:4001/news/">News</a>

                  <ul class="dropdown">
                    

                      

                      <li><a  href="http://localhost:4001/news/archive/">News Archive</a></li>
                    
                  </ul>

              </li>
              <li class="divider"></li>
            
          
        

              

          
          

            
            
              <li class="active"><a  href="http://localhost:4001/software/">Resources</a></li>
              <li class="divider"></li>

            
            
          
        

              

          
          
        
        
      </ul>
    </section>
  </nav>
</div><!-- /#navigation -->

	

	

<div id="masthead">
	<div class="row">
		<div class="small-4 columns"></div>
		<div class="small-8 columns">
			<a id="logo" href="http://localhost:4001/" title="The Jackson Lab – ">
				<img src="http://localhost:4001/assets/img/Site_Logo4.png" alt="The Jackson Lab – ">
			</a>
		</div><!-- /.small-12.columns -->
	</div><!-- /.row -->
</div><!-- /#masthead -->











	


<div class="row t30">
	<div class="medium-12 columns">
		<article>
			<header>
				<p class="subheadline"></p>
				<h1>Software</h1>
			</header>

			
			<p class="teaser">
				
			</p>
			

			<div data-magellan-expedition="fixed">
  <ul class="sub-nav">
    <li data-magellan-arrival="CIViC"><a href="#CIViC">CIViC</a></li>
    <li data-magellan-arrival="pVACtools"><a href="#pVACtools">pVACtools</a></li>
    <li data-magellan-arrival="DGIdb"><a href="#DGIdb">DGIdb</a></li>
    <li data-magellan-arrival="GenVisR"><a href="#GenVisR">GenVisR</a></li>
    <li data-magellan-arrival="RegTools"><a href="#RegTools">RegTools</a></li>
    <li data-magellan-arrival="TrioMix"><a href="#TrioMix">TrioMix</a></li>
    <li data-magellan-arrival="VAtools"><a href="#VAtools">VAtools</a></li>
    <li data-magellan-arrival="OpenCAP"><a href="#OpenCAP">OpenCAP</a></li>
    <li data-magellan-arrival="DoCM"><a href="#DoCM">DoCM</a></li>
    <li data-magellan-arrival="CIViCpy"><a href="#CIViCpy">CIViCpy</a></li>

  </ul>
</div>

<div class="row">
    <div class="small-12 columns">
        <h2 data-magellan-destination="CIViC"><a href="http://civicdb.org"> CIViC: Clinical Interpretations of Variants in Cancer </a></h2>
        <a name="CIViC"></a>
    </div>
</div>

<div class="row">
    <div class="small-6 columns">
        
          <img src="/assets/img/software/civic_screenshot.png" />
        
    </div>
    <div class="small-6 columns">
        
          <a href="http://civicdb.org"><img src="/assets/img/software/CIViC_logo.png" /></a>
        
        <h5>URL: <a href="http://civicdb.org">civicdb.org</a></h5>
        <h5>Description:</h5>
        <p>
            CIViC provides a knowledgebase and curation system for expert crowdsourcing the clinical interpretation of variants in cancer.
        </p>
        <h5>Team members:</h5>
        <p>
            Malachi Griffith, Nicholas Spies, Kilannin Krysiak, Josh McMichael, Adam Coffman, Arpad Danos, Susanna Kiwala, Ben Ainscough, Erica Barnell, Alex Wagner, Zachary Skidmore, Robert Lesurf, Lee Trani, Avinash Ramu, Katie Campbell, Ajay Venigalla, Lana Sheta, Obi Griffith
        </p>

        <h5>Source code:</h5>
          
          <a href="https://github.com/griffithlab/civic-client">civic-client (GitHub)</a>
          

          
          <br /><a href="https://github.com/griffithlab/civic-server">civic-server (GitHub)</a>
          

        
        <h5>Citation(s):</h5>
          1. <a href="https://www.ncbi.nlm.nih.gov/pubmed/28138153">Griffith, Spies, Krysiak et al. 2017. CIViC is a community knowledgebase for expert crowdsourcing the clinical interpretation of variants in cancer. Nature Genetics. 49(2):170–174.</a>
        

          

          
 
    </div>
</div>
<hr />

<div class="row">
    <div class="small-12 columns">
        <h2 data-magellan-destination="pVACtools"><a href="http://pvactools.org"> pVACtools: Tools for Personalized Cancer Vaccine Design </a></h2>
        <a name="pVACtools"></a>
    </div>
</div>

<div class="row">
    <div class="small-6 columns">
        
          <img src="/assets/img/software/pVACtools_overview.png" />
        
    </div>
    <div class="small-6 columns">
        
          <a href="http://pvactools.org"><img src="/assets/img/software/pvactools_logo.png" /></a>
        
        <h5>URL: <a href="http://pvactools.org">pvactools.org</a></h5>
        <h5>Description:</h5>
        <p>
            A computational toolkit for aiding design of personalized cancer vaccines.
        </p>
        <h5>Team members:</h5>
        <p>
            Susanna Kiwala, Huiming Xia, Megan Richters, Joshua McMichael, Christopher Miller, Jasreet Hundal, Yang-Yang Feng, Aaron Graubert, Alex Wollam, Amber Wollam, Connor Liu, Jason Walker, Obi Griffith, Malachi Griffith
        </p>

        <h5>Source code:</h5>
          
          <a href="https://github.com/griffithlab/pVACtools">pVACtools (GitHub)</a>
          

          

        
        <h5>Citation(s):</h5>
          1. <a href="https://www.ncbi.nlm.nih.gov/pubmed/26825632">Hundal et al. 2016. pVAC-Seq: A genome-guided in silico approach to identifying tumor neoantigens. Genome Medicine. 8(1):11.</a>
        

          
          <br />2. <a href="https://www.ncbi.nlm.nih.gov/pubmed/31907209">Hundal, Kiwala et al. 2020. pVACtools: A Computational Toolkit to Identify and Visualize Cancer Neoantigens. Cancer Immunology Research. 8(3):409-420.</a>
          

          
 
    </div>
</div>
<hr />

<div class="row">
    <div class="small-12 columns">
        <h2 data-magellan-destination="DGIdb"><a href="http://dgidb.org"> DGIdb: The Drug-Gene Interaction Database </a></h2>
        <a name="DGIdb"></a>
    </div>
</div>

<div class="row">
    <div class="small-6 columns">
        
          <img src="/assets/img/software/dgidb_screenshot.png" />
        
    </div>
    <div class="small-6 columns">
        
          <a href="http://dgidb.org"><img src="/assets/img/software/dgidb_logo.png" /></a>
        
        <h5>URL: <a href="http://dgidb.org">dgidb.org</a></h5>
        <h5>Description:</h5>
        <p>
            DGIdb offers user-friendly browsing, searching, and filtering of information on drug-gene interactions and the druggable genome, mined from over thirty trusted sources.
        </p>
        <h5>Team members:</h5>
        <p>
            Kelsy Cotto, Alex Wagner, Yang-Yang Feng, Susanna Kiwala, Adam Coffman, Gregory Spies, Alex Wollam, Nicholas Spies, Obi Griffith, Malachi Griffith
        </p>

        <h5>Source code:</h5>
          
          <a href="https://github.com/griffithlab/dgi-db">dgi-db (GitHub)</a>
          

          

        
        <h5>Citation(s):</h5>
          1. <a href="https://www.ncbi.nlm.nih.gov/pubmed/24122041">Griffith, Griffith, et al. 2013. DGIdb: mining the druggable genome. Nature Methods. 10(12):1209-10.</a>
        

          
          <br />2. <a href="https://www.ncbi.nlm.nih.gov/pubmed/26531824">Wagner et al. 2016. DGIdb 2.0: mining clinically relevant drug-gene interactions. Nucleic Acids Research. 44(D1):D1036-44.</a>
          

          
          <br />3. <a href="https://www.ncbi.nlm.nih.gov/pubmed/29156001">Cotto, Wagner et al. 2018. DGIdb 3.0: a redesign and expansion of the drug-gene interaction database. Nucleic Acids Research.</a>
          
 
    </div>
</div>
<hr />

<div class="row">
    <div class="small-12 columns">
        <h2 data-magellan-destination="GenVisR"><a href="https://bioconductor.org/packages/release/bioc/html/GenVisR.html"> GenVisR: Genomic Visualizations in R </a></h2>
        <a name="GenVisR"></a>
    </div>
</div>

<div class="row">
    <div class="small-6 columns">
        
          <img src="/assets/img/software/genvisr_overview.png" />
        
    </div>
    <div class="small-6 columns">
        
        <h5>URL: <a href="https://bioconductor.org/packages/release/bioc/html/GenVisR.html">github.com/griffithlab/GenVisR</a></h5>
        <h5>Description:</h5>
        <p>
            GenVisR is a bioconductor package designed to allow users to produce highly customizable publication quality graphics for genomic data primarily at the cohort level.
        </p>
        <h5>Team members:</h5>
        <p>
            Zachary Skidmore, Alex Wagner, Katie Campbell, Kelsy Cotto, Jason Kunisaki, Obi Griffith, Malachi Griffith.
        </p>

        <h5>Source code:</h5>
          
          <a href="https://bioconductor.org/packages/release/bioc/html/GenVisR.html">GenVisR (Bioconductor)</a>
          

          
          <br /><a href="https://github.com/griffithlab/GenVisR">GenVisR (GitHub)</a>
          

        
        <h5>Citation(s):</h5>
          1. <a href="https://www.ncbi.nlm.nih.gov/pubmed/27288499">Skidmore et al. 2016. GenVisR: Genomic Visualizations in R. Bioinformatics. 32(19):3012-4.</a>
        

          
          <br />2. <a href="https://pubmed.ncbi.nlm.nih.gov/34506690">Skidmore et al. 2021. Exploring the Genomic Landscape of Cancer Patient Cohorts with GenVisR. Current Protocols.</a>
          

          
 
    </div>
</div>
<hr />

<div class="row">
    <div class="small-12 columns">
        <h2 data-magellan-destination="RegTools"><a href="http://regtools.org"> RegTools </a></h2>
        <a name="RegTools"></a>
    </div>
</div>

<div class="row">
    <div class="small-6 columns">
        
          <img src="/assets/img/software/regtools_overview.png" />
        
    </div>
    <div class="small-6 columns">
        
        <h5>URL: <a href="http://regtools.org">regtools.org</a></h5>
        <h5>Description:</h5>
        <p>
            RegTools is a set of tools that integrate DNA-seq and RNA-seq data to help interpret mutations in a regulatory and splicing context.
        </p>
        <h5>Team members:</h5>
        <p>
            Yang-Yang Feng, Avinash Ramu, Kelsy Cotto, Jason Kunisaki, Zachary Skidmore, Obi Griffith, Malachi Griffith
        </p>

        <h5>Source code:</h5>
          
          <a href="https://github.com/griffithlab/regtools">RegTools (GitHub)</a>
          

          

        

          

          
 
    </div>
</div>
<hr />

<div class="row">
    <div class="small-12 columns">
        <h2 data-magellan-destination="VAtools"><a href="http://vatools.org"> VCF Annotation Tools </a></h2>
        <a name="VAtools"></a>
    </div>
</div>

<div class="row">
    <div class="small-6 columns">
        
          <img src="/assets/img/software/vatools_screenshot.png" />
        
    </div>
    <div class="small-6 columns">
        
          <a href="http://vatools.org"><img src="/assets/img/software/vatools_logo2.png" /></a>
        
        <h5>URL: <a href="http://vatools.org">vatools.org</a></h5>
        <h5>Description:</h5>
        <p>
            VAtools is a python package that includes several tools to annotate VCF files with data from other tools.
        </p>
        <h5>Team members:</h5>
        <p>
            Susanna Kiwala
        </p>

        <h5>Source code:</h5>
          
          <a href="https://github.com/griffithlab/VAtools">VAtools (GitHub)</a>
          

          

        
        <h5>Citation(s):</h5>
          1. <a href="https://github.com/griffithlab/VAtools">In preparation</a>
        

          

          
 
    </div>
</div>
<hr />

<div class="row">
    <div class="small-12 columns">
        <h2 data-magellan-destination="TrioMix"><a href="http://triomix.io/"> TrioMix </a></h2>
        <a name="TrioMix"></a>
    </div>
</div>

<div class="row">
    <div class="small-6 columns">
        
          <img src="/assets/img/software/triomix.png" />
        
    </div>
    <div class="small-6 columns">
        
        <h5>URL: <a href="http://triomix.io/">triomix.io</a></h5>
        <h5>Description:</h5>
        <p>
            TrioMix is a software tool for detecting intrafamilial DNA contamination, chimerism, and uniparental disomy from sequencing family trios.
        </p>
        <h5>Team members:</h5>
        <p>
            Chris Yoon, Obi Griffith, Malachi Griffith
        </p>

        <h5>Source code:</h5>
          
          <a href="https://github.com/cjyoon/triomix">TrioMix (GitHub)</a>
          

          

        
        <h5>Citation(s):</h5>
          1. <a href="https://doi.org/10.1101/gr.276794.122">Yoon et al. 2022. Estimation of intrafamilial DNA contamination in family trio genome sequencing using deviation from Mendelian inheritance. Genome Research (2022).</a>
        

          

          
 
    </div>
</div>
<hr />

<div class="row">
    <div class="small-12 columns">
        <h2 data-magellan-destination="OpenCAP"><a href="http://opencap.org"> OpenCAP: Blueprints for Precision Medicine </a></h2>
        <a name="OpenCAP"></a>
    </div>
</div>

<div class="row">
    <div class="small-6 columns">
        
          <img src="/assets/img/software/opencap_screenshot.png" />
        
    </div>
    <div class="small-6 columns">
        
          <a href="http://opencap.org"><img src="/assets/img/software/opencap_logo2.png" /></a>
        
        <h5>URL: <a href="http://opencap.org">opencap.org</a></h5>
        <h5>Description:</h5>
        <p>
            An open resource for building custom capture panels that are transparently linked to clinical variant annotations.
        </p>
        <h5>Team members:</h5>
        <p>
            Erica Barnell, Adam Waalkes, Kelsi Penewit, Katie Campbell, Zachary Skidmore, Colin Pritchard, Todd Fehniger, Ravindra Uppaluri, Ramaswamy Govindan, Malachi Griffith, Stephen Salipante, Obi Griffith
        </p>

        <h5>Source code:</h5>
          
          <a href="https://github.com/griffithlab/civic-panel">OpenCAP (GitHub)</a>
          

          

        
        <h5>Citation(s):</h5>
          1. <a href="https://www.ncbi.nlm.nih.gov/pubmed/31618044">Barnell et al. 2019. Open-Sourced CIViC Annotation Pipeline to Identify and Annotate Clinically Relevant Variants Using Single-Molecule Molecular Inversion Probes. JCO Clin Cancer Inform. 3:1-12.</a>
        

          

          
 
    </div>
</div>
<hr />

<div class="row">
    <div class="small-12 columns">
        <h2 data-magellan-destination="DoCM"><a href="http://docm.info"> DoCM: Database of Curated Mutations </a></h2>
        <a name="DoCM"></a>
    </div>
</div>

<div class="row">
    <div class="small-6 columns">
        
          <img src="/assets/img/software/docm_screenshot.png" />
        
    </div>
    <div class="small-6 columns">
        
          <a href="http://docm.info"><img src="/assets/img/software/docm_logo.png" /></a>
        
        <h5>URL: <a href="http://docm.info">docm.info</a></h5>
        <h5>Description:</h5>
        <p>
            The Database of Curated Mutations (DoCM) is an open access resource that outlines literature supported somatic mutations that are clinically or biologically relevant.
        </p>
        <h5>Team members:</h5>
        <p>
            Ben Ainscough, Malachi Griffith, Adam Coffman, Alex Wagner, Jason Kunisaki, Joshua McMichael, Obi Griffith
        </p>

        <h5>Source code:</h5>
          
          <a href="https://github.com/griffithlab/docm">docm (GitHub)</a>
          

          

        
        <h5>Citation(s):</h5>
          1. <a href="https://www.ncbi.nlm.nih.gov/pubmed/27684579">Ainscough et al. 2016. DoCM: a database of curated mutations in cancer. Nature Methods. 13(10):806-7.</a>
        

          

          
 
    </div>
</div>
<hr />

<div class="row">
    <div class="small-12 columns">
        <h2 data-magellan-destination="CIViCpy"><a href="http://civicpy.org"> CIViCpy: A software development kit (SDK) for the CIViC knowledgebase </a></h2>
        <a name="CIViCpy"></a>
    </div>
</div>

<div class="row">
    <div class="small-6 columns">
        
          <img src="/assets/img/software/civicpy_workflow.png" />
        
    </div>
    <div class="small-6 columns">
        
          <a href="http://civicpy.org"><img src="/assets/img/software/civicpy_logo.png" /></a>
        
        <h5>URL: <a href="http://civicpy.org">civicpy.org</a></h5>
        <h5>Description:</h5>
        <p>
            CIViCpy is designed to be an intuitive SDK and analysis toolkit for the Clinical Interpretations of Variants in Cancer (CIViC) knowledgebase.
        </p>
        <h5>Team members:</h5>
        <p>
            Alex Wagner, Susanna Kiwala, Adam Coffman, Joshua McMichael, Kelsy Cotto, Thomas Mooney, Erica Barnell, Kilannin Krysiak, Arpad Danos, Jason Walker, Obi Griffith, Malachi Griffith
        </p>

        <h5>Source code:</h5>
          
          <a href="https://github.com/griffithlab/civicpy">CIViCpy (GitHub)</a>
          

          

        
        <h5>Citation(s):</h5>
          1. <a href="https://ascopubs.org/doi/pdf/10.1200/CCI.19.00127">Wagner and Kiwala, et al. 2020. CIViCpy: A Python Software Development and Analysis Toolkit for the CIViC Knowledgebase. JCO Clin Cancer Inform. 4:245-253.</a>
        

          

          
 
    </div>
</div>
<hr />


		</article>
	</div><!-- /.medium-12.columns -->
</div><!-- /.row -->




	
	<div id="up-to-top" class="row">
  <div class="small-12 columns" style="text-align: right;">
    <a class="iconfont" href="#top-of-page">&#xf108;</a>
  </div><!-- /.small-12.columns -->
</div><!-- /.row -->


<footer id="footer-content" class="bg-grau">
  <div id="footer">
    <div class="row">
      <div class="medium-6 large-6 columns">
        <h5 class="shadow-black">About This Site</h5>
        <p class="shadow-black">
          This is the website for the academic lab of Malachi and Obi Griffith
          <a href="http://localhost:4001/">More ›</a>
        </p>
        <section id="subfooter-left" class="small-12 medium-6 columns">
          <ul class="inline-list social-icons">
          
            <li><a href="https://github.com/TheJacksonLab" target="_blank" class="fa-brands fa-github" title="Github repo for site."></a></li>
          
            <li><a href="" target="_blank" class="" title=""></a></li>
          
            <li><a href="" target="_blank" class="" title=""></a></li>
          
            <li><a href="" target="_blank" class="" title=""></a></li>
          
            <li><a href="" target="_blank" class="" title=""></a></li>
          
            <li><a href="" target="_blank" class="" title=""></a></li>
          
            <li><a href="" target="_blank" class="" title=""></a></li>
          
            <li><a href="" target="_blank" class="" title=""></a></li>
          
            <li><a href="" target="_blank" class="" title=""></a></li>
          
            <li><a href="" target="_blank" class="" title=""></a></li>
          
            <li><a href="" target="_blank" class="" title=""></a></li>
          
            <li><a href="" target="_blank" class="" title=""></a></li>
          
            <li><a href="" target="_blank" class="" title=""></a></li>
          
            <li><a href="" target="_blank" class="" title=""></a></li>
          
            <li><a href="" target="_blank" class="" title=""></a></li>
          
          </ul>
        </section>
      </div>
      <!-- /.large-6.columns -->
      <div class="medium-6 large-6 columns">
        <a href="https://chemistry.illinois.edu/"><img src="/assets/img/Illinois.png"></a>
      </div><!-- /.large-6.columns -->
    </div><!-- /.row -->
    <div class="row">
        <section id="subfooter-right" class="medium-6 large-centered columns">
          <p>Powered by <a href="http://jekyllrb.com/" target="_blank">Jekyll</a> based on the theme <a href="http://phlow.github.io/feeling-responsive/">Feeling Responsive</a>.</p>
        </section>
    </div>
  </div><!-- /#footer -->
</footer>

	

	


<script src="http://localhost:4001/assets/js/javascript.min.js"></script>


<script>
$(document).ready(function() {
  // Foundation should have auto-initialized, but ensure magellan-expedition is working
  if ($('[data-magellan-expedition]').length > 0) {
    // Wait for Foundation to be ready, then ensure magellan is initialized
    setTimeout(function() {
      // Check if magellan-expedition is initialized
      var expedition = $('[data-magellan-expedition]');
      expedition.each(function() {
        var $exp = $(this);
        // If not initialized, initialize it
        if (!$exp.data('magellan-expedition-init')) {
          // Try to initialize via Foundation
          try {
            $(document).foundation('magellan-expedition', 'init', $exp);
          } catch(e) {
            // If that fails, Foundation should have auto-initialized
            console.log('Magellan initialization check:', e);
          }
        }
      });
      // Force reflow and trigger events
      $(document).foundation('magellan-expedition', 'reflow');
      $(window).trigger('scroll.fndtn.magellan');
      $(window).trigger('resize.fndtn.magellan');
    }, 300);
  }
});
</script>



<script>
    $("#masthead").backstretch("http://localhost:4000/images/genvis-dna-bg_optimized_v1a.png", {transition: 'fade', transitionDuration: 2000});
    $("#masthead-with-text").backstretch("http://localhost:4000/images/genvis-dna-bg_optimized_v1a.png", {transition: 'fade', transitionDuration: 2000});
</script>











</body>
</html>

